Arm/joint.hpp

```diff
@@ -89,6 +83,16 @@ class Joint
   }
 
  private:
+ 
+  float ReturnChangedIfZero(float acceleration)
+  {
+    if (acceleration == 0)
+    {
+      acceleration = 0.001;
+    }
+    return acceleration;
+  }
```

Comment:
I notice we have a few repeated functions in both joint.hpp and wrist_joint.hpp. Might be worth creating a base class in the future to remove duplicated logic.

Reply:
maybe we make joint.hpp be the base and make arm joint logic be arm_joint.hpp?

Reply:
I was starting to think the same thing lol, will do.